core/modules/qdisp/testQDisp.cc

```diff
@@ -63,48 +63,67 @@ class ChunkMsgReceiverMock : public MsgReceiver {
 
 /** Simple functor for testing that _retryfunc has been called.
  */
-class RetryTest : public util::VoidCallable<void> {
+class JobQueryTest : public qdisp::JobQuery {
 public:
-    typedef std::shared_ptr<RetryTest> Ptr;
-    RetryTest() : _retryCalled(false) {}
-    virtual ~RetryTest() {}
-    virtual void operator()() {
-        _retryCalled = true;
-        LOGF_INFO("_retryCalled=%1%" % _retryCalled);
+    typedef std::shared_ptr<JobQueryTest> Ptr;
+    JobQueryTest(qdisp::Executive* executive,
+              qdisp::JobDescription const& jobDescription,
+              qdisp::JobStatus::Ptr jobStatus,
+              qdisp::MarkCompleteFunc::Ptr markCompleteFunc)
+       : qdisp::JobQuery(executive, jobDescription, jobStatus, markCompleteFunc),
```

Comment:
i was talking about pushing the last 3 arguments 4 spaces to the right to align them with the 1st one. I guess I should have inserted my comment one line higher. I have nothing against the line starting with colon.